### DISCUSSION TITLE
Hi i create team, but i cancle it already but still get charge money for it

### DISCUSSION THREAD

**Discussion by DaoHuyTuan** [vercel/community#5335] [Account Issue]
### Summary

Hi i create team, but i cancle it already but still get charge money for it, can i get the refund of it please, i'm very urgent 

### Steps to Reproduce

Hi i create team, but i cancle it already but still get charge money for it, can i get the refund of it please, i'm very urgent

**Comment by amyegan** [comment]
Hi @DaoHuyTuan. We can't safely discuss your account and billing information in public. Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team with `Invoice Enquiry` selected as the Problem Area. That will get you in contact with the team that can help with billing issues 🙇
